src/Pickles/Pickles.TestFrameworks.UnitTests/MsTest/WhenParsingMsTestResultsFileWithIgnoredExample.cs

```diff
@@ -0,0 +1,65 @@
+﻿//  --------------------------------------------------------------------------------------------------------------------
+//  <copyright file="WhenParsingMsTestResultsFileWithEmptyExampleValues.cs" company="PicklesDoc">
+//  Copyright 2011 Jeffrey Cameron
+//  Copyright 2012-present PicklesDoc team and community contributors
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//  </copyright>
+//  --------------------------------------------------------------------------------------------------------------------
+
+
+using NUnit.Framework;
+using PicklesDoc.Pickles.ObjectModel;
+using PicklesDoc.Pickles.TestFrameworks.MsTest;
+using System.Collections.Generic;
+using NFluent;
+
+namespace PicklesDoc.Pickles.TestFrameworks.UnitTests.MsTest
+{
+    [TestFixture]
+    public class WhenParsingMsTestResultsFileWithIgnoredExample : StandardTestSuite<MsTestResults>
+    {
+        public WhenParsingMsTestResultsFileWithIgnoredExample()
+            : base("MsTest." + "results-example-mstest-ignoredexample.trx")
```

Comment:
Aha. I got you now. I'm just writing the test from the perspective of how a real life scenario will be (where you will be having an Example which is ignored and which resulted in the example results not  to exists. The example results does not go missing by itself unless it was ignored or someone updated the .feature and then trying to generate the doc (with .trx that is ran with older .feature). I believe the first scenario is more likely to happen, thus that explain the test